projects/addon-commerce/components/input-card-grouped/input-card-grouped.template.html

```diff
@@ -23,11 +24,11 @@
         </span>
         <label>
             <span
-                *ngIf="cardNumberTexts$ | async as texts"
+                *ngIf="texts"
```

Comment:
I think it was here just to declare async result, no need for this ngIf. Same for the below ngIfs.